### ISSUE TITLE
ib_srp-backport on OEL 6.9

### ISSUE THREAD

**Issue by grafol1** [bvanassche/ib_srp-backport#8]
hi,

I have been using the backport successfully with our ION products for years now. with the latest upgrade of our servers to OEL 6.8 and OEL 6.9 I am facing the problem, that all SRP commands are throwing the following errors

[root@ ~]# ibsrpdm
ibv_reg_mr failed
Resource allocation failed

the kernel version is

 4.1.12-61.1.25.el6uek.x86_64 #2 SMP Wed Jan 11 19:13:03 PST 2017 x86_64 x86_64 x86_64 GNU/Linux

srp tools

rpm -qa | grep srp
srptools-1.0.3-2.el6.x86_64
ib_srp-backport-4.1.12-61.1.25.el6uek.x86_64-2.0.41-1.x86_64

the built-in infiniband stack is being used.

libibumad-1.3.9-1.el6.x86_64

any ideas?

thank you.

Oliver Graf.

**Comment by bvanassche** [comment]
Hello Oliver,
The ib_srp-backport driver is not involved at all when ibsrpdm queries SRP targets. ibsrpdm sends MAD (management datagrams) to the subnet manager and SRP target systems to query target information. So if you are using the ibsrpdm executable provided by Oracle and that executable fails this is unrelated to the ib_srp-backport driver. Have you already considered to contact Oracle support?

**Comment by grafol1** [comment]
Classification: For internal use only
Hi Bart,

Thank you for your reply. Yes, I have opened an SR with Oracle and the feedback was to install the oracle-ofed package. this is based on the ofed 2.X package, which is several years old and brings back all the issues we had 4 years ago. the srptools package is based on version 0.40, including all the issues I faced many years ago.

Thank you.


Mit freundlichen Grüssen

Dipl.-Phys. Oliver Graf
Vice President | Global Unix ES
In-house Unix & VMS Operations
IT Infrastructure Operations

Deutsche Bank AG
COO - Group CIO Safety and Soundness
Alfred-Herrhausen-Allee 16-24, 65760 Frankfurt am Main, Germany
Tel. +49(69)910-31522
Mobile +49 177 4925807
Email oliver-a.graf@db.com<mailto:gerald-thomas.stuetz@db.com>
